js/compiler/igor-paths.js

```diff
@@ -2,6 +2,9 @@
 import { join_path } from '../GMConstructor.js';
 import { Err } from '../utils/Err.js';
 
+const process = require('node:process');
```

Comment:
Second instance of `require`, also I think we already have `process` somewhere, or did earlier...

Reply:
I saw the init.js script before, and considered it while doing this for igor-paths, but decided to use `require` here because this exports `igor_platform_map` directly, instead of being in a function or something like that, and I can't really pass arguments into it that way. VSCode doesn't complain to me aout mixing `import` and `require` (and yes it does about type-related things so it is working), though I'm not sure how you use the TS language server.
I'll see what I could do.